From ed805f6992f3c5cfb78f139a5e49ab62909fb944 Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Tue, 3 Mar 2015 20:53:43 +0000
Subject: [PATCH] configure: Fix giflib check

As libgif often links to libX11, and some linker flags require that
dependent libraries must be explicitly linked with, try adding -lX11 to
the linker commandline when trying to link with -lgif.

https://bugzilla.gnome.org/show_bug.cgi?id=745582
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 11c785e..fb4364d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2102,7 +2102,7 @@ AC_ARG_ENABLE(libgif,
 
 if test "x$enable_libgif" != "xno" ; then
    AC_CHECK_HEADER(gif_lib.h,
-   AC_CHECK_LIB(gif, DGifOpen))
+   AC_CHECK_LIB(gif, DGifOpen, [], [], [-lX11]))
 
    have_libgif=${ac_cv_lib_gif_DGifOpen:-no}
 
-- 
2.3.1

